### PR TITLE
[ICDS Dashboard] UI location filter sorting ALL issue

### PR DESCRIPTION
**Pull request by challabeehyv**
##### SUMMARY
Removed sorting the location filters in the UI

##### FEATURE FLAG
no

### CODE REVIEW

**Comment by challabeehyv** [comment]
Screenshot from 2019-12-27 15-54-43

File: custom/icds_reports/static/js/angular-services/locations.service.js [hunk 1]
```diff
@@ -117,11 +117,7 @@ window.angular.module('icdsApp').factory('locationsService', ['$http', '$locatio
                 if (!selectedLocation || selectedLocation.location_id === 'all') {
                     return [];
                 }
-                return _.sortBy(
```
Comment (Rohit25negi) [inline custom/icds_reports/static/js/angular-services/locations.service.js]: will this not stop sorting all together? we want to sort actual location names

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
left a question

File: custom/icds_reports/static/js/angular-services/locations.service.js (hunk 1 above)
Comment (challabeehyv) [reply custom/icds_reports/static/js/angular-services/locations.service.js]: they are getting sorted in the backend. Don't think elements will be added to it runtime in the UI to sort

**Review by Rohit25negi — APPROVED** [review]

**Pull request merged**